pkg/exchange/coinbase/exchage.go

```diff
@@ -308,25 +310,36 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 		log.Warnf("limit %d is greater than the maximum limit 300, set to 300", options.Limit)
 		options.Limit = DefaultKLineLimit
 	}
-	granity := interval.String()
+	granity := fmt.Sprintf("%d", interval.Seconds())
```

Comment:
use `string(interval) `?

Reply:
`string(interval)` will be `"1m"` but the Coinbase API is expecting `"60"`.

Reply:
how about renaming the varialbe "granity" to "granularity"?

Reply:
Opps, my bad.
Mispelling.
Will fix it.

Reply:
Done.